src/services/priceService.ts

```diff
@@ -219,7 +222,7 @@ export default class PriceService {
     const tokenBPrices =
       coins && coins[`${defillamaId}:${tokenB.address}`] && coins[`${defillamaId}:${tokenB.address}`].prices;
 
-    const tokenAIsBaseToken = STABLE_COINS.includes(tokenA.symbol);
+    const tokenAIsBaseToken = STABLE_COINS.includes(tokenA.symbol) || !STABLE_COINS.includes(tokenB.symbol);
```

Comment:
This was defined in the test cases for `getPriceForGraph`:
`'when no token is a stable coin, it should return the prices based on token A'`
But it wasn't implemented in the actual function that way! So if the token A is not a stable coin, we were always setting price based on token B

Reply:
USDC.e isn't recognized as a stablecoin because of its symbol, which messes up the graph data for this case. I think we should just drop this `||` condition and the related test case. We could add more conditions, but it seems like a lot for such a small thing.